src/Input/Input.js

```diff
@@ -6,7 +6,7 @@ import classNames from 'classnames';
 import { createStyleSheet } from 'jss-theme-reactor';
 import customPropTypes from '../utils/customPropTypes';
 
-function isDirty(obj) {
+export function isDirty(obj) {
```

Comment:
@mbrookes My motivation for sharing it was regarding the past history of Material-UI, we have changed that logic in the past on the `master` branch, the logic is deeply linked in the two component. I want to avoid inconsistency.